app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -19,8 +19,13 @@ def sign_class(widget)
     widget.percentual_change > 0 ? 'u-plus' : 'u-minus'
   end
 
-  def dashboard_collection_link(singular_name, collection, path, plural = nil)
-    link_to pluralize(number_to_human(collection.size), singular_name, plural), path, class: 'DashboardNavigation-link'
+  def dashboard_collection_link(singular_name, collection, path, plural = nil, icon_name = nil)
+    link_to path, class: 'DashboardNavigation-link' do
+      link = ''
+      link << icon(icon_name) + ' ' if icon_name
```

Comment:
This will be forbidden in future ruby or with `frozen_string_literal: true`

Reply:
yes, rightly so, it's ugly as hell

Reply:
I'm also considering refactoring this method to use named parameters. would be next iteration